### ISSUE TITLE
fix accessibility flags for site

### ISSUE THREAD

**Issue by nick-mon1** [18F/analytics.usa.gov#1116]
Run ANDI, AxeTools, pa11y to address accessibility errors and warnings for:

- [x] home page
- [x] data page
- [x] test on mac
- [ ] test on pc
- [ ] 508 team could provide a11y testing

## Resources

- https://www.dhs.gov/trusted-tester
- https://www.section508.gov/content/guide-accessible-web-design-development/

## What To Test

Use [these metrics as top 3 priority](https://itdashboard.gov/faq) a11y fixes and [18F Checklist](https://guides.18f.gov/accessibility/landmarks/):

- color contrast
- markup errors
- images with alt text
- keyboard tabbing

## How I Tested

- [x] [pa11y](https://github.com/pa11y/pa11y)
- [x] ANDI
- [x] AxeDevtools

## Critical Fixes

- [x] color contrast meets minimum 4.5:1 ratio
- [x] tab order follows predictable pattern
- [x] tab focus is always visible
- [x] tabbing order cycles back to address bar
- [x] no modal functionality
- [x] no hidden content
- [x] search input form has aria-label for screen reader
- [x] select dropdown is keyboard friendly, can focus and use keys to select agency
- [x] select dropdown arrow is not critical for user, decorative image. Only image used in site design
   
## Less Critical

- [x] all elements should exist within a landmark
- [x] top pages tabs refactored for screenreaders
- [x] color contrast is WCAG consistent
- [x] heading structure follows a logical hierarchy
- [x] tables have caption and row and column scopes for screen readers to read tabled data

## Less Critical, not fixed 

Moved to https://github.com/18F/analytics.usa.gov/issues/1141

- [ ] Run **ANDI** and fix `Ambiguous Link: same name/description as another link but different href.`
- [ ] Run **pa11y** and fix `This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard`
- [ ] Run **pa11y** and fix `Duplicate id attribute value "_fed_an_ua_tag" found on the web page`, duplicate `id` is being set by two script tags.
